src/Orleans.Streaming.Redis/Providers/Streams/RedisStreamBuilder.cs

```diff
@@ -56,17 +60,18 @@ public class ClusterClientRedisStreamConfigurator : ClusterClientPersistentStrea
         public ClusterClientRedisStreamConfigurator(string name, IClientBuilder builder)
             : base(name, builder, RedisQueueAdapterFactory.Create)
         {
-            this.clientBuilder.ConfigureApplicationParts(parts => parts.AddFrameworkPart(typeof(RedisQueueAdapterFactory).Assembly))
-                 .ConfigureServices(services =>
-                 {
-                     services.TryAddSingleton(SilentLogger.Logger);
-                     services.ConfigureNamedOptionForLogging<RedisStreamOptions>(name);
-                     services.TryAddSingleton(CachedConnectionMultiplexerFactory.Default);
-                     services.TryAddSingleton<ISerializationManager, OrleansSerializationManager>();
-                     services.AddSingleton<IRedisDataAdapter, RedisDataAdapter>();
-                     services.AddTransient<IConfigurationValidator>(sp => new RedisStreamOptionsValidator(sp.GetOptionsByName<RedisStreamOptions>(name), name));
-                     services.ConfigureNamedOptionForLogging<HashRingStreamQueueMapperOptions>(name);
-                 });
+            builder
```

Comment:
Didn't want to pull out the IClientBuilder abstraction and replace with Action<>'s like above?

Reply:
I would have thought so too, but no. The base class `ClusterClientPersistentStreamConfigurator` requires IClientBuilder.